lib/maplibre-compose/src/androidMain/kotlin/dev/sargunv/maplibrecompose/core/AndroidMap.kt

```diff
@@ -273,27 +277,25 @@ internal class AndroidMap(
     offset: DpOffset,
     layerIds: Set<String>?,
     predicate: Expression<Boolean>?,
-  ): List<Feature> =
-    map
-      .queryRenderedFeatures(
-        offset.toPointF(density),
-        predicate?.toMLNExpression(),
-        *layerIds.orEmpty().toTypedArray(),
-      )
+  ): List<Feature> {
+    // Kotlin hack to pass null to a java nullable varargs
+    val query: (PointF, MLNExpression?, Array<String>?) -> List<MLNFeature> =
+      map::queryRenderedFeatures
```

Comment:
Passing null instead of empty array in order to align with a future fix to: https://github.com/maplibre/maplibre-native/issues/2828

(it also just looks weird to an unfamiliar reader that an empty array would be the same as naming every layer)

But since Kotlin doesn't support Java's nullable varargs, we have to eliminate the varargs with the soft cast in `val query: ...`.